test/e2e/performanceprofile/functests/1_performance/performance.go

```diff
@@ -445,105 +445,6 @@ var _ = Describe("[rfe_id:27368][performance]", Ordered, func() {
 		})
 	})
 
-	Context("KubeletConfig experimental annotation", func() {
```

Comment:
@ffromani Do you remember why we removed this? I see it is part of the original patch too.

Reply:
yes: https://github.com/openshift/cluster-node-tuning-operator/pull/854/commits/a623c50e0ea8dd322a7b319731eccde68f966678

Reply:
+1